patches/openwrt/0007-ramips-mt7620-fix-tp-link-archer-c50v1-wlan-leds.patch

```diff
@@ -0,0 +1,27 @@
+From 2fa666e15c4b6ca977bcf40da62bbc6f1fcfedfb Mon Sep 17 00:00:00 2001
+From: Dark4MD <github.web@manu.li>
+Date: Thu, 7 Nov 2019 09:46:25 +0100
+Subject: [PATCH] ramips-mt7620: fix TP-Link Archer C50 v1 WLAN LEDs
+
+---
+ target/linux/ramips/base-files/etc/board.d/01_leds | 4 ++--
+ 1 file changed, 2 insertions(+), 2 deletions(-)
+
+diff --git a/target/linux/ramips/base-files/etc/board.d/01_leds b/target/linux/ramips/base-files/etc/board.d/01_leds
+index f5e914c518..464bfb78aa 100755
+--- a/target/linux/ramips/base-files/etc/board.d/01_leds
++++ b/target/linux/ramips/base-files/etc/board.d/01_leds
+@@ -83,8 +83,8 @@ c20i)
+ c50)
+ 	ucidef_set_led_switch "lan" "lan" "$boardname:green:lan" "switch0" "0x1e"
+ 	ucidef_set_led_switch "wan" "wan" "$boardname:green:wan" "switch0" "0x01"
+-	ucidef_set_led_netdev "wlan2g" "wlan2g" "$boardname:green:wlan2g" wlan1
+-	set_wifi_led "$boardname:green:wlan5g"
++	ucidef_set_led_wlan "wlan2g" "wlan2g" "$boardname:green:wlan2g" "phy1tx"
```

Comment:
Apart from the fact you should send this patch upstream - Why you specifically chose the tx trigger here? This creates an inconsistency from the other WiFi LED.

Reply:
The wifi driver for the 2,4ghz soc radio doesn't have an tpt trigger. Would it be better for both to use tx then or something different?

I will send it upstream then, as soon as i know what would be the best option.

Reply:
Well ,then the trigger has to be implemented into the Ralink WiFi driver.
Otherwise, this device is broken as it currently is, as radio status as well as transmission has to be indicated.

Reply:
Marked as broken, removed Patch.